site/content/entry/citra-progress-report-2017-august.md

```diff
@@ -0,0 +1,176 @@
++++
+date = "2017-09-04T15:07:00-04:00"
+title = "Citra Progress Report - 2017 August"
+tags = [ "progress-report" ]
+author = "anodium"
+coauthor = "saphiresurf"
+forum = 0000
++++
+
+2017 has been an amazing year, with more work having been put into the project
+than ever before, but it's not over yet! Last we met was
+[June](/entry/citra-progress-report-2017-june/), and just two months later the
+[Citra issue tracker](https://github.com/citra-emu/citra/pulls) is brimming with
+lots of changes once more. I am extremely excited for this month (and what's coming
+up the next few months) but we're getting ahead of ourselves! On this progress
+report, let's check out the big fish in the July and August pond of patches!
+
+## Updating The Software Renderer ([This](https://github.com/citra-emu/citra/pull/2766), [that](https://github.com/citra-emu/citra/pull/2822), [here](https://github.com/citra-emu/citra/pull/2871), [there](https://github.com/citra-emu/citra/pull/2872), and [those](https://github.com/citra-emu/citra/pull/2891)) by [wwylele](https://github.com/wwylele) and [Subv](https://github.com/Subv)
+
+Citra has two main rendering backends, software and OpenGL, but until very
+recently, no one gave much attention to the software backend. The OpenGL backend
+is faster, made better use of the GPU, and allows things such as texture forwarding
+for higher resolution rendering. But there is one thing on which the OpenGL backend
+falls flat on its face --- accuracy.
```

Comment:
Don't think you need three dashes - just one will do.

Reply:
Typically an em-dash (—) without spaces before and after it is used in these circumstances to emphasize a concluding sentence (a colon is generally used instead when emphasis isn't important).

e.g.

"But there is one thing on which the OpenGL backend falls flat on its face—accuracy"